lib/decisionmaking-flow.js

```diff
@@ -150,7 +151,26 @@ WHERE {
   return [];
 }
 
+
+async function getLatestSubcase(decisionmakingFlowUri) {
+  const queryString = `
+    PREFIX dct: <http://purl.org/dc/terms/>
+    PREFIX dossier: <https://data.vlaanderen.be/ns/dossier#>
+
+    SELECT ?uri ?title {
```

Comment:
Add `DISTINCT` here (it's just good practice to have it, even if you're only selecting 1 row.